src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import StyleObjectPropType from '../../prop-types/style';
+import TabsNav from './TabsNav';
+import TabsPanes from './TabsPanes';
+import TabsPane from './TabsPane';
+import TabsTab from './TabsTab';
+
+let lastInstanceId = 0;
+
+const propTypes = {
+    selectedIndex: PropTypes.number,
```

Comment:
Could we sort this prop list alphabetically like the other components please :)

Reply:
Oh, fixed!